libretro_core_options.h

```diff
@@ -602,7 +602,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(enable_memcard1),
       "Enable Memory Card 1 (Restart Required)",
       NULL,
-      "Select whether to emulate a second memory card in slot 1. When disabled, games can only access the memory card in slot 0. Note: Some games require this option to be disabled for correct operation (e.g. Codename Tenka). Note: Memory Card 1 uses the Mednafen (.mcr) save format.",
+      "Emulate a second memory card in slot 1. When disabled, games can only access the memory card in slot 0. Note: Some games require this option to be disabled for correct operation (e.g. Codename Tenka). Note: Memory Card 1 uses the Mednafen (.mcr) save format.",
```

Comment:
Lowercase the second "Memory Card" to unify all cases. Seems that Sony didn't trademark the words "Memory Card", so they don't require mandatory uppercasing.